### PR TITLE
add chroot-mode, so it works without qemu/kernel

### PR DESCRIPTION
**Pull request by bittorf**
rebased intop of 98058e6b70ba4e28839c0b4898d7049c172ea0d4

### CODE REVIEW

File: rootfs.sh [hunk 1]
```diff
@@ -98,8 +99,9 @@ get_file() {
     else
 	bname=$(basename "$url" ".tar.${ext}")
     fi
-    cp "${bname}.kaem" tmp/after
-    cp "../sources/$(basename "$url")" tmp/after
+
+    [ -f "${bname}.kaem" ] && cp -v "${bname}.kaem" tmp/after
+    cp -v "../sources/$(basename "$url")" tmp/after
```
Comment (stikonas) [inline rootfs.sh:104]: you can drop this now. I've reworked this part of `get_file` function.

File: rootfs.sh [hunk 2]
```diff
@@ -119,13 +121,18 @@ cd tmp
 find . | cpio -H newc -o | gzip > initramfs.igz
 
 # Run
-${QEMU_CMD:-qemu-system-x86_64} -enable-kvm \
-    -m "${RAM:-8G}" \
-    -nographic \
-    -no-reboot \
-    -kernel ../../kernel -initrd initramfs.igz -append console=ttyS0
+if [ "${QEMU_CMD}" = 'chroot' ]; then
+	sudo PATH="/after/bin:${PATH}" chroot . /init
```
Comment (stikonas) [inline rootfs.sh:125]: Maybe add a comment that this is a bug in old version of kaem but is not required with new kaem?

File: rootfs.sh (hunk 1 above)
Comment (bittorf) [reply rootfs.sh:103]: but your change does not work/breaks.

IMHO better is:
```
for FILE in "${base}."*; do test -f "$FILE" && cp "$FILE" tmp/after; done
```
What do you think?

File: rootfs.sh (hunk 2 above)
Comment (bittorf) [reply rootfs.sh:124]: if this is not needed anymore, i can just drop die PATH export.
are you sure?

**Comment by fosslinux** [comment]
Merged.

**Pull request closed without merging**